lua_source_ddf.proto missing from defoldsdk.zip

**Describe the bug (REQUIRED)**
[lua_source_ddf.proto](https://github.com/defold/defold/blob/dev/engine/script/src/script/lua_source_ddf.proto) is missing from the defoldsdk.zip downloads.

**To Reproduce (REQUIRED)**
1. Download defoldsdk.zip from one of the latest releases (e.g. [the one from 1.8.0](https://github.com/defold/defold/releases/download/1.8.0/defoldsdk.zip))
2. Extract it
3. Try to compile the .proto files

**Expected behavior (REQUIRED)**
It should include all protobuf definitions.

**Defold version (REQUIRED):**
Tested just now with 1.8.0 and 1.7.0. I remember this happening with previous versions too.

**Logs (OPTIONAL):**
```
$ cd defoldsdk/share/proto
$ protoc --python_out=../py ./**/*

script/lua_source_ddf.proto: File not found.
gameobject/properties_ddf.proto:4:1: warning: Import ddf/ddf_extensions.proto is unused.
gameobject/properties_ddf.proto:5:1: warning: Import ddf/ddf_math.proto is unused.
gameobject/lua_ddf.proto:7:1: Import "script/lua_source_ddf.proto" was not found or had errors.
gameobject/lua_ddf.proto:18:14: "LuaSource" is not defined.
engine/engine_ddf.proto:7:1: Import "gameobject/lua_ddf.proto" was not found or had errors.
engine/engine_ddf.proto:26:14: "dmLuaDDF.LuaModule" is not defined.
```

**Workaround (OPTIONAL):**
Manually downloading lua_source_ddf.proto or using the precompiled header files in the SDK.

--- a/engine/script/src/wscript
+++ b/engine/script/src/wscript
@@ -78,8 +78,7 @@ def build(bld):
          target = '__init__.py',
          shell = True)
 
-    bld.install_files('${PREFIX}/share/script', 'script/lua_source_ddf.proto script/ddf_script.proto')
-    bld.install_files('${PREFIX}/share/proto/script', 'script/ddf_script.proto script/sys_ddf.proto')
+    bld.install_files('${PREFIX}/share/proto/script', 'script/ddf_script.proto script/sys_ddf.proto script/lua_source_ddf.proto')
     bld.install_files('${PREFIX}/lib/python/script', '__init__.py')
 
     if 'web' in bld.env.PLATFORM:

